tools/oeedger8r/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oeedger8r",
+  "version": "0.6.0",
+  "description": "The Open Enclave SDK's oeedger8r",
+  "license": "MIT",
+
+  "esy": {
+    "build": "dune build -p #{self.name}",
+    "buildsInSource": "_build",
+    "release": {
+      "bin": "oeedger8r",
+      "includePackages": [
+        "root"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "@opam/dune": "~1.11.3",
+    "ocaml": "~4.6.0"
+  },
```

Comment:
If we need to update ocaml (or dune/merlin/ocamlformat), we can update the version here in this file.